stock_getter/backend_stockgetter/stock_service/tests.py

```diff
@@ -1,3 +1,64 @@
 from django.test import TestCase
+from django.urls import reverse, resolve
+from django.test import Client
+from .views import StockListServiceAPIView, StockDataServiceAPIView
+import requests
+
+
+class TestUrls(TestCase):
+
+    def test_stock_list_url_is_valid(self):
+        url = reverse('stock_service:stock_list')
+        self.assertEqual(resolve(url).func.view_class, StockListServiceAPIView)
+
+    def test_stock_data_url_is_valid(self):
+        url = reverse('stock_service:stock_data')
+        self.assertEqual(resolve(url).func.view_class, StockDataServiceAPIView)
+
+
+class TestStockListServiceAPIView(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.data_url = reverse('stock_service:stock_list')
+
+    def test_response_stock_list(self):
+        """This is method that checks the response of stock_list
+        making sure it's equal to 200 (site found)
+        """
+        response = self.client.get(self.data_url)
+        # status 200, can be found on internet.
+        self.assertEquals(response.status_code, 200)
+        self.assertIsNotNone(response)
+
+
+class TestStockDataServiceAPIView(TestCase):
+    def setUp(self):
+
+        self.stock_data_daily_url = \
+            'https://www.alphavantage.co/query?function=TIME_' \
```

Comment:
There is a lot of repetition here. Can we extract this as template and use something like string format to input DAILY/WEEKLY/MONTHLY?